### PR TITLE
docs: update README with IdP information

### PR DESCRIPTION
**Pull request by ssvegaraju**
Issue #, if available:

Description of changes:
Added some more information about the OAuth2 IdP.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by rsmayda — APPROVED** [review]
PR looks good for me -- please wait for Brandi's approval prior to merging

**Review by brandroid-tw — APPROVED** [review]

**Pull request merged**
